scripts/gui/microbe_editor.mjs

```diff
@@ -310,6 +319,22 @@ function onExitToMenuClickedEditor(){
     }
 }
 
+function updateGuiButtons(isNucleusPresent){
```

Comment:
This function needs to also block the player from clicking the buttons. Perhaps the button click code should check if it contains `DisabledButton` class and if it does then ignore the click.